### ISSUE TITLE
Docker image 6.0 causes warning "vm.max_map_count is too low"

### ISSUE THREAD

**Issue by Hermholtz** [docker-library/mongo#575]
I've pulled the default MongoDB 6.0 image and created a container. It starts successfully, but when connecting via `mongosh` it shows the following information:
```
   The server generated these startup warnings when booting
   2022-10-12T21:11:29.967+00:00: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine. See http://dochub.mongodb.org/core/prodnotes-filesystem
   2022-10-12T21:11:31.718+00:00: Access control is not enabled for the database. Read and write access to data and configuration is unrestricted
   2022-10-12T21:11:31.719+00:00: vm.max_map_count is too low
```
The last warning is what bothers me. The others will be fixed at my side.

In docker logs there's:
```
{"t":{"$date":"2022-10-10T22:09:47.184+00:00"},"s":"W",  "c":"CONTROL",  "id":5123300, "ctx":"initandlisten","msg":"vm.max_map_count is too low","attr":{"currentValue":65530,"recommendedMinimum":1677720,"maxConns":838860},"tags":["startupWarnings"]}
```

I tried adjusting the value of `vm.max_map_count` in the operating system via `sysctl -w` but no luck.

Does it have to be set in the container? If so, can you make the adjustments and publish an image that doesn't show this warning?

Thank you.

**Comment by yosifkit** [comment]
Since the value is host global (i.e. not namespaced) it cannot be changed in the container and it reflects in the container as soon as it is changed from the host. So it should be seen by `mongod` when started after the change.

**Comment by Hermholtz** [comment]
That indeed helped, I did not enter sufficient value on my first attempt.

I was able to get rid of the warning with `sysctl -w vm.max_map_count=1677720`, but then I have recreated my container with saner limit of the concurrent connections from the default 838.860 to just 1000 (as my container is only used by one backend). For somebody looking for an advice, the argument is:
```
docker run -d -v /path/to/persistent/storage --name mongo-test --restart always mongo:6.0 --maxConns 1000
```
Pay attention to the `--maxConns 1000`.

After lowering maxConns the `sysctl` was not required anymore and could be reverted to OS default.
